src/ok.rs

```diff
@@ -605,3 +671,209 @@ fn memlock_rlimit() -> TestResult {
         mesg: Some(mesg),
     }
 }
+
+fn msr_bit_read(reg:u32, cpu:u16) -> Result<u64,anyhow::Error> {
+    let mut msr = Msr::new(reg, cpu).context("Error Reading MSR")?;
+    let raw_value = msr.read()?;
+    Ok(raw_value)
+}
+
+fn sme_test() -> TestResult{
+        let raw_value = match msr_bit_read(0xC0010010,0) {
+            Ok(raw) => raw,
+            Err(e) => return TestResult{
+                        name: "SME Test".to_string(),
```

Comment:
```suggestion
                        name: "SME Enabled".to_string(),
```

Reply:
Don't change output message, keep consistent for different test results. you could even create a variable at the top of the function with the test name and pass that so that you don't have to worry about keeping track of names